### ISSUE TITLE
Text/input box is overlap

### ISSUE THREAD

**Issue by keai4le** [zarunbal/LogExpert#263]
# Bug
Some text overlaps in the settings UI/search bar
See #171 for the screenshot.

- LogExpert Version
All, but with the **high-dpi** version, only the search bar has this issue
- Windows Version
10.0.22622.586
- Local file or Network
NA
- Used Columnizer
No
- Is this version specific or a new problem
All version

- Screenshot
See #171

**Comment by nitin065** [comment]
Similar issue in the "Highlighting and action triggers" section. Can't see Save button. Tried expanding but didn't work. I tried to use the keyboard, though the navigation was not proper but it somehow blindly clicked save.

**Comment by Hirogen** [comment]
> Similar issue in the "Highlighting and action triggers" section. Can't see Save button. Tried expanding but didn't work. I tried to use the keyboard, though the navigation was not proper but it somehow blindly clicked save.

could you post a screenshot, there shouldn't be a save button, but maybe we talk about different dialogs

![image](https://user-images.githubusercontent.com/8225398/188854970-3c2add30-7fcb-485c-a06d-ccf43e192e85.png)

**Comment by nitin065** [comment]
> could you post a screenshot, there shouldn't be a save button, but maybe we talk about different dialogs

It was an "Ok" button, I thought it could be "Save" as I was not able to see it.

After waking up the PC from sleep mode, I don't see the issue anymore. So, it is working as expected for me now.

**Comment by nitin065** [comment]
![image](https://user-images.githubusercontent.com/20437084/189578357-35f76b08-f4b8-43ae-8124-0fecfcc315ee.png)

I somehow got the issue again. Cannot say how exactly it is getting reproduced.

**Comment by Hirogen** [comment]
Yes that is a DPI Issue, since the "normal" Version is not DPI-Aware only the High-DPI Version is DPI Aware and should scale accordingly! 
Could you please use the HIGH-DPI Version (https://github.com/zarunbal/LogExpert/releases/tag/v1.9.0.High.DPI) its the same as the 1.9.0 just for High DPI and High DPI Screens

**Comment by Hirogen** [comment]
> # Bug
> Some text overlaps in the settings UI/search bar See #171 for the screenshot.
>   See [Some text/input is covered #171](https://github.com/zarunbal/LogExpert/issues/171)

could you please add the resolution of your screen, I encountered this with a machine via VNC and a resolution oaf 1024x768

**Comment by Hirogen** [comment]
with the High DPI Version this should work, but there is an exception, on screens that have a lower height then 936 pxls (thats the minimum height of the Highlight Dialog), the Dialog will be cropped and not be fully visible. On those screens pressing ENTER should activate the "OK" button!

**Comment by Hirogen** [comment]
fixed with => https://github.com/zarunbal/LogExpert/tree/High-DPI-Test
